src/OpenTelemetry/CHANGELOG.md

```diff
@@ -19,6 +19,9 @@
   Previously it would return `null`.
   ([#4885](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4885))
 
+* Updated `Microsoft.Extensions.Logging` package version to `8.0.0-rc.1.23419.4`.
+  ([#4920](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4920))
+
```

Comment:
@cijothomas I think we should remove this.

If you look at SDK it only depends on `Microsoft.Extensions.Logging.Configuration` and gets all the logging dependencies transitively through that:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/d4d5122316814e8e8f3670e7ce8a8d981d0c42a5/src/OpenTelemetry/OpenTelemetry.csproj#L9

So this PR bumping `Microsoft.Extensions.Logging` actually won't do anything to SDK!

Reply:
Alright.... Then should we not bump Ms.Ext.Log.Config instead to 8.0?

Reply:
> Alright.... Then should we not bump Ms.Ext.Log.Config instead to 8.0?

Not in this PR.